hint.py

```diff
@@ -0,0 +1,57 @@
+import pygame
```

Comment:
You should add a short version of the license used in the activity as others files have.